src/css/grid.css

```diff
@@ -1,26 +1,18 @@
-.row,
-.column {
+.row {
   display: flex;
   justify-content: space-between;
   align-items: stretch;
-  margin-left: -1em;
-  margin-right: -1em;
-}
-
-.row {
+  margin-left: -2em;
```

Comment:
Maybe use a variable here and below (`.row > *`) to ensure they are always synced